src/components/ScreenView/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React, { ReactElement } from "react";
+import { SafeAreaView, StatusBar } from "react-native";
+
+interface ScreenView {
+  children: ReactElement;
+}
+
+export const ScreenView = ({ children }: ScreenView): ReactElement => (
```

Comment:
Don't need to explicitly define the interface if we turn off the prop-types linting. `export const ScreenView: FunctionComponent = ({ children }) => (`

Reply:
Ah great, this works